beta-src/src/enums/Season.ts

```diff
@@ -0,0 +1,9 @@
+enum Season {
+  SPRING = "Spring",
+  SPRING_RETREATS = "Spring Retreats",
+  FALL = "Fall",
+  FALL_RETREATS = "Fall Retreats",
+  WINTER = "Winter",
```

Comment:
`SPRING_RETREATS` and `FALL_RETREATS` are not seasons. Also, pull the latest staging and check the enums folder as there may be seasons already defined.